rust/src/tests/chmod.rs

```diff
@@ -141,3 +142,82 @@ enoent_comp_test_case!(
 );
 
 enoent_symlink_named_file_test_case!(chmod(~path, Mode::empty()));
+
+// TODO: Lot of OS-dependent code, should we add a guard to skip OSes for which the EFTYPE trigger is unknown?
```

Comment:
Good question.  I think the way you did it is fine.  Skipping the test on OSes like Linux would be ok too.  But this is probably best, because it's what the old pjdfstest did.